### PR TITLE
Bug 1277017 - Added checking if spec.DockerImageRepository and spec.Tags are not empty

### PR DESCRIPTION
**Pull request by soltysh**
Fixes https://bugzilla.redhat.com/show_bug.cgi?id=1277017

@miminar ptal

### CODE REVIEW

File: pkg/cmd/cli/cmd/importimage.go [hunk 1]
```diff
@@ -83,6 +83,9 @@ func RunImportImage(f *clientcmd.Factory, out io.Writer, cmd *cobra.Command, arg
 			Spec:       imageapi.ImageStreamSpec{DockerImageRepository: from},
 		}
 	} else {
+		if len(stream.Spec.DockerImageRepository) == 0 && len(stream.Spec.Tags) == 0 {
+			return fmt.Errorf("only image streams with spec.dockerImageRepository or spec.Tags set may have images imported")
```
Comment (miminar) [inline pkg/cmd/cli/cmd/importimage.go]: `s/Tags/tags/`

**Comment by miminar** [comment]
Just a cosmetic nitpick. Otherwise LGTM.

**Comment by soltysh** [comment]
Nit fixed, @openshift/approvers ptal

**Comment by soltysh** [comment]
@smarterclayton updated

**Comment by smarterclayton** [comment]
LGTM [merge]

**Pull request merged**